libraries/scrcpy/src/base/control-message-type.ts

```
@@ -1,24 +1,7 @@
-    UHidCreate: 12,
-    UHidInput: 13,
-    UHidDestroy: 14,
-    OpenHardKeyboardSettings: 15,
-    StartApp: 16,
-    ResetVideo: 17,
-} as const;
+import * as ScrcpyControlMessageType from "./control-message-type-value.js";
 
-export type ScrcpyControlMessageType =
+// These IDs change between versions, so always use `options.getControlMessageTypes()`
+type ScrcpyControlMessageType =
```

Biome (2.1.2)

[error] Shouldn't redeclare 'ScrcpyControlMessageType'. Consider to delete it or rename it.

'ScrcpyControlMessageType' is defined here:


(lint/suspicious/noRedeclare)